helm-charts/basehub/templates/network-policy-dask-worker.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: dask-worker-network-policy
+spec:
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/name: dask-gateway
+      app.kubernetes.io/component: dask-worker
+  policyTypes:
+    - Egress
+  egress:
+    # Allow access to everything except private IPs
+    - to:
+        - ipBlock:
+            cidr: 0.0.0.0/0
+            except:
+              - 10.0.0.0/8
+              - 172.16.0.0/12
+              - 192.168.0.0/16
+    # Allow access to DNS
+    - ports:
+        - port: 53
+          protocol: UDP
+        - port: 53
+          protocol: TCP
+      to:
+        - namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: kube-system
+          podSelector:
+            matchLabels:
+              k8s-app: kube-dns
```

Comment:
Leave a comment here that while this says `kube-dns`, it also works with core-dns (what is actually used by EKS) because it has that as a label